nncf/quantization/algorithms/weight_compression/openvino_backend.py

```diff
@@ -348,28 +350,29 @@ def _get_bitwidth_distribution_str(all_weight_params: List[WeightNodeParams]) ->
     return pretty_string
 
 
-def _assign_mixed_precision(all_weight_params: List[WeightNodeParams], ratio: float, group_size: int) -> None:
+def _assign_mixed_precision(
+    all_weight_params: List[WeightNodeParams], ratio: float, primary_config: WeightCompressionConfig
+) -> None:
     """
     Assigns mixed quantization scheme (e.g. uniform int8 or non-uniform nf4) for weights based on some criteria.
 
     :param all_weight_params: List of information about each weight node. The quantization scheme is added to this info.
-    :param ratio: the ratio between baseline and backup precisions (e.g. 0.9 means 90% of layers quantized to NF4
+    :param ratio: The ratio between primary and backup precisions (e.g. 0.9 means 90% of layers quantized to NF4
         and the rest to INT8).
-    :param group_size: number of weights (e.g. 128) in the channel dimension that share quantization parameters (scale).
-        The value -1 means no grouping.
+    :param primary_config: Information on how to compress (quantize) weights to primary precision.
+    :return: None.
     """
-    nf4_config = WeightCompressionConfig(num_bits=4, is_nf4=True, group_size=group_size)
     if ratio != 1:
```

Comment:
Should we consider to use an eps to compare float values or is it ok?

Reply:
![image](https://github.com/openvinotoolkit/nncf/assets/4014476/b596a05e-5bd0-4c48-ac65-4a20a1e1dfea)
nobody will set 10 or more digits after a comma. It's superfluous here.